src/Orchard.Cms.Web/Modules/Orchard.Demo/Orchard.Demo.csproj

```diff
@@ -20,6 +20,7 @@
     <ProjectReference Include="..\..\..\OrchardCore\Orchard.Environment.Navigation\Orchard.Environment.Navigation.csproj" />
     <ProjectReference Include="..\..\..\OrchardCore\Orchard.ResourceManagement\Orchard.ResourceManagement.csproj" />
     <ProjectReference Include="..\..\..\OrchardCore\Orchard.ResourceManagement.Abstractions\Orchard.ResourceManagement.Abstractions.csproj" />
+    <ProjectReference Include="..\Orchard.Contents\Orchard.Contents.csproj" />
```

Comment:
I mean in authorization server tenant you need to enable modules for which you want to have permissions available to assign a role. In this way it would be enough to enable the WhateverModule.Abstration module instead of the WhateverModule